Script/dungeon.dat

```diff
@@ -95,6 +94,8 @@ Team BETRAYED_TEAM;
   KillEvilness = 10;
 }
 
+/* Dungeons to be included in the game */
+
 Include "dungeons/Attnam.dat";
```

Comment:
Yes, but it adds some non-cross-platform code (or an external library dependency, e.g. boost::filesystem). I'll implement this quickly.